src/Illuminate/Cache/CacheManager.php

```diff
@@ -251,7 +251,7 @@ public function repository(Store $store)
 	 */
 	protected function getPrefix(array $config)
 	{
-		return array_get($config, 'prefix') ?: $this->app['config']['cache.prefix'];
+		return array_get($config, 'prefix', $this->app['config']['cache.prefix']);
```

Comment:
Except for this, it might be intentional to just use `isset(..) ? .. : ..` for speed/performance.